content/en/policies/other/annotation_lifetime.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Enforce annotation lifetime"
+category: Sample
+version: 1.4.2
+subject: Pod
+policyType: "validate"
+description: >
+    Pod lifetime annotation can be no greater than 8 hours.
+---
+
+## Policy Definition
+<a href="https://github.com/kyverno/policies/raw/main//other/pod_lifetime_annotation/policy.yaml" target="-blank">/other/pod_lifetime_annotation/policy.yaml</a>
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: pod-lifetime
+  annotations:
+    policies.kyverno.io/title: Enforce pod duration
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/minversion: 1.4,2
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Pod lifetime annotation can be no greater than 8 hours.
+spec:
+  background: false
+  validationFailureAction: enforce
```

Comment:
We haven't yet put this in a style guide for kyverno/policies, but we'd like to have all (or almost all) `validate` policies set `validationFailureAction=audit` out of more of a safety function for unsuspecting users. You could just hand-edit this in this MD file and make a similar change to kyverno/policies to make it easier than rerendering.

Reply:
Switched this to `audit`